src/itemActionTitle/ItemActionTitle.story.mdx

```diff
@@ -0,0 +1,71 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { BubbleIcon } from '../icon/bubbleIcon'
+import { ItemActionTitle } from './index'
+
+<Meta title="Items/ItemActionTitle" />
+
+# ItemActionTitle
+
+### Button
+
+<Canvas>
+  <Story name="Button">
+    <ItemActionTitle
+      titleLabel="Title: primary information"
+      subLabel="Subtitle: secondary information"
+      action="Action"
+    />
+  </Story>
+</Canvas>
+
+### Link with leftAddon
+
+<Canvas>
+  <Story name="Link witn leftAddon">
+    <ItemActionTitle
+      titleLabel="Title: primary information"
+      subLabel="Subtitle: secondary information"
+      action="Action"
+      leftAddon={<BubbleIcon />}
+      href="#"
+    />
+  </Story>
+</Canvas>
+
+## Specifications
+
+### Normal state
+
+These items launch modals. We use them when the action is not part of the main flow.
+
+### Behaviour
+
+Like all ITEMS, the whole zone of the component is clickable.
+
+### Long texts
+
+There’s no limitation on the number of lines for the title and additional info.
+
+### Rule to go on the second line
+
+The minimum spacing between the Title and the Action text is 24px.
+As soon as both elements cannot respect this 24px, the Action text goes on the second line.
+
+## Usage
+
+```jsx
+import { ItemActionTitle } from '@blablacar/ui-library/build/itemActionTitle'
+
+<ItemActionTitle
+  titleLabel="Primary info"
```

Comment:
The API shoud follow the very same `Item` **API**. 
We struggle identifying `MainInfo`, `Title`, `titleLabel`... and sometimes we have to map data differently because the API isn't consistent

Reply:
That's good point. Ideally we should follow the Pixar naming IMHO. Which is "title" in this case
![image](https://user-images.githubusercontent.com/2495124/100598267-ca2d9900-32fe-11eb-8486-c0c045a031fe.png)

Reply:
I think @gabrielagabriel suggest to follow 
```
leftTitle={leftTitle}
leftBody={leftBody}
rightBody={rightBody}
```
instead of
```
leftTitle={titleLabel}
leftBody={subLabel}
rightBody={action}
```

Reply:
So I followed the  `item` API